OpenTibia/Assets/Scripts/Core/Components/SidebarWidgetBottomResizer.cs

```diff
@@ -91,7 +91,7 @@ public class SidebarWidgetBottomResizer : BasicElement,
             _mouseOverResizer = false;
 
             // if no drag is occuring and cursor has been pushed, then pop it
-            if (!_mouseHeldDown && _cursorPushed)
+            if (!_mouseHeldDown && !OpenTibiaUnity.InputHandler.IsAnyMousePressed() && _cursorPushed)
```

Comment:
Looks good assuming there is no way any of these: `OpenTibiaUnity.InputHandler.IsAnyMousePressed()` can return null :)

Reply:
That's right. There is no way one of those could return null.